### PR TITLE
draft of new roles post

### PR DESCRIPTION
**Pull request by mondus**
Draft of the post for our new roles using the advert from the open ended one. There are some issues to resolve before merging and the links current use the Sheffield jobs site links as a result.

- [ ] The fixed term role does not have a contract end date of end of march as agreed.
- [ ] The ATJ for the fixed term role is an older one than the open ended position.
- [ ] HR have not added the fixed term role to jobs.ac.uk
- [ ] Open ended role is fixed term on jobs.ac.uk

Other things to do
 - [x] Advertise open ended role on society vacancies board
 - [ ]  Advertise fixed term role on society vacancies board
 - [ ]  Advertise both roles on UKRSE #jobs

### CODE REVIEW

File: _posts/2021-11-12-rse-roles.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+slug: 2021-11-12-rse-roles
+date: 2021-11-12 16:30:00 UTC
+tags:
+category:
+link:
+description:
+type: text
+---
+
+Are you interested in leading a team to develop and facilitate reproducible software through collaborations across the whole spectrum of university research areas?
```
Comment (bobturneruk) [inline _posts/2021-11-12-rse-roles.md]: Leading?

File: _posts/2021-11-12-rse-roles.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Educated to PhD level (close to completion, or equivalent experience), you will have a track record of high-quality software development and a commitment to open, reproducible research. Applications are welcome from both software development generalists and from RSEs with inter-disciplinary technology focused skills which either complement or extend our existing portfolio, such as data analytics (including statistical computing, machine learning and deep learning), numerical computing, High-Performance Computing (HPC; including multi-core and GPUs), etc.
+
+You will enhance the university’s capability and expertise in developing research software as part of academic research and will collaborate closely with IT Services to ensure the RSE team aspects of the university’s overarching Research IT strategy.  The RSE team of 13 at the University of Sheffield is led by an EPSRC Research Software Engineering Fellow and is guided by a steering panel of senior university staff who are committed to supporting the role of RSEs within the research community.
```
Comment (bobturneruk) [inline _posts/2021-11-12-rse-roles.md]: Are you still a fellow?

File: _posts/2021-11-12-rse-roles.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+The post provides an excellent opportunity for research software skills development. You will actively support the wider research community and will be encouraged to represent the team in national/international activities. For example, current team members hold positions and qualifications including: Society of RSE Trustee, rOpenSci Editor, NVIDIA Deep Learning Institute accredited trainers, and accredited Carpentries organisation instructors.
```
Comment (bobturneruk) [inline _posts/2021-11-12-rse-roles.md]: Do we have any carpentries instructors? I'm a maintainer. And a reviewer for JOSS. Anna's reprohack position?

File: _posts/2021-11-12-rse-roles.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+tags:
+category:
+link:
+description:
+type: text
+---
+
+Are you interested in leading a team to develop and facilitate reproducible software through collaborations across the whole spectrum of university research areas?
+
+An opening is available to join the growing Research Software Engineering (RSE) team on an open-ended contract. The role is ideal for someone with a passion for research and for good software engineering looking to use their skills in collaborative software development and reproducible research to make an impact on research projects across the whole of the University of Sheffield and beyond.
```
Comment (bobturneruk) [inline _posts/2021-11-12-rse-roles.md]:
```suggestion
There are openings available to join the growing Research Software Engineering (RSE) team on an open-ended contract or fixed term contract. The role is ideal for someone with a passion for research and for good software engineering looking to use their skills in collaborative software development and reproducible research to make an impact on research projects across the whole of the University of Sheffield and beyond.
```

File: _posts/2021-11-12-rse-roles.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+A key responsibility of the role will be embedding software engineering skills through collaboration and training/teaching. Applicants should have experience of collaborating with key stakeholders/clients to specify, develop, improve and deliver technical software and the ability to undertake such collaborations in a research environment. You will work within the existing RSE team across a range of research projects, departments, and institutes within the university, as well as advocating good practice, providing free at the point of use support and assisting in the development of new strategic collaborations.
+
+Educated to PhD level (close to completion, or equivalent experience), you will have a track record of high-quality software development and a commitment to open, reproducible research. Applications are welcome from both software development generalists and from RSEs with inter-disciplinary technology focused skills which either complement or extend our existing portfolio, such as data analytics (including statistical computing, machine learning and deep learning), numerical computing, High-Performance Computing (HPC; including multi-core and GPUs), etc.
```
Comment (bobturneruk) [inline _posts/2021-11-12-rse-roles.md]:
```suggestion
Educated to PhD level (close to completion, or equivalent experience), you will have a track record of high-quality software development (using a research relevant language such as R, Python, MATLAB, C, C++) and a commitment to open, reproducible research. Applications are welcome from both software development generalists and from RSEs with inter-disciplinary technology focused skills which either complement or extend our existing portfolio, such as data analytics (including statistical computing, machine learning and deep learning), numerical computing, High-Performance Computing (HPC; including multi-core and GPUs), etc.
```

**Review by bobturneruk — APPROVED** [review]
Some comments and suggestions.

File: _posts/2021-11-12-rse-roles.md (hunk 1 above)
Comment (mondus) [inline _posts/2021-11-12-rse-roles.md]:
```suggestion
Are you interested in joining a team to develop and facilitate reproducible software through collaborations across the whole spectrum of university research areas?
```

File: _posts/2021-11-12-rse-roles.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+You will enhance the university’s capability and expertise in developing research software as part of academic research and will collaborate closely with IT Services to ensure the RSE team aspects of the university’s overarching Research IT strategy.  The RSE team of 13 at the University of Sheffield is led by an EPSRC Research Software Engineering Fellow and is guided by a steering panel of senior university staff who are committed to supporting the role of RSEs within the research community.
```
Comment (mondus) [inline _posts/2021-11-12-rse-roles.md]:
```suggestion
You will enhance the university’s capability and expertise in developing research software as part of academic research and will collaborate closely with IT Services to ensure the RSE team aspects of the university’s overarching Research IT strategy.  The RSE team of 13 at the University of Sheffield is led by a former EPSRC Research Software Engineering Fellow and is guided by a steering panel of senior university staff who are committed to supporting the role of RSEs within the research community.
```

File: _posts/2021-11-12-rse-roles.md (hunk 2 above)
Comment (mondus) [reply _posts/2021-11-12-rse-roles.md]:
```suggestion
You will enhance the university’s capability and expertise in developing research software as part of academic research and will collaborate closely with IT Services to ensure the RSE team aspects of the university’s overarching Research IT strategy.  The RSE team of 13 at the University of Sheffield is led by a former EPSRC Research Software Engineering Fellow and is guided by a steering panel of senior university staff who are committed to supporting the role of RSEs within the research community.
```
A former one now....

File: _posts/2021-11-12-rse-roles.md (hunk 3 above)
Comment (mondus) [reply _posts/2021-11-12-rse-roles.md]:
```suggestion
The post provides an excellent opportunity for research software skills development. You will actively support the wider research community and will be encouraged to represent the team in national/international activities. For example, current team members hold positions and qualifications including: membership of the Society of RSE, rOpenSci Editor, reviewers for JOSS, NVIDIA Deep Learning Institute accredited trainers, and Software Carpentries maintainers.
```

Also removed trustees

File: _posts/2021-11-12-rse-roles.md (hunk 1 above)
Comment (mondus) [reply _posts/2021-11-12-rse-roles.md]: joining

File: _posts/2021-11-12-rse-roles.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+tags:
+category:
+link:
+description:
+type: text
+---
+
+Are you interested in joining a team to develop and facilitate reproducible software through collaborations across the whole spectrum of university research areas?
+
+There are openings available to join the growing Research Software Engineering (RSE) team on an open-ended contract or fixed term contract. The role is ideal for someone with a passion for research and for good software engineering looking to use their skills in collaborative software development and reproducible research to make an impact on research projects across the whole of the University of Sheffield and beyond.
```
Comment (bobturneruk) [inline _posts/2021-11-12-rse-roles.md]:
```suggestion
There are openings available to join the growing Research Software Engineering (RSE) team on an open-ended contract or fixed term contract. The RSE role is ideal for someone with a passion for research and for good software engineering looking to use their skills in collaborative software development and reproducible research to make an impact on research projects across the whole of the University of Sheffield and beyond.
```

File: _posts/2021-11-12-rse-roles.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+Educated to PhD level (close to completion, or equivalent experience), you will have a track record of high-quality software development (using a research relevant language such as R, Python, MATLAB, C, C++) and a commitment to open, reproducible research. Applications are welcome from both software development generalists and from RSEs with inter-disciplinary technology focused skills which either complement or extend our existing portfolio, such as data analytics (including statistical computing, machine learning and deep learning), numerical computing, High-Performance Computing (HPC; including multi-core and GPUs), etc.
+
+You will enhance the university’s capability and expertise in developing research software as part of academic research and will collaborate closely with IT Services to ensure the RSE team aspects of the university’s overarching Research IT strategy.  The RSE team of 13 at the University of Sheffield is led by a former EPSRC Research Software Engineering Fellow and is guided by a steering panel of senior university staff who are committed to supporting the role of RSEs within the research community.
```
Comment (bobturneruk) [inline _posts/2021-11-12-rse-roles.md]:
```suggestion
You will enhance the university’s capability and expertise in developing research software as part of academic research and will collaborate closely with IT Services to ensure the delivery of RSE team aspects of the university’s overarching Research IT strategy.  The RSE team of 13 at the University of Sheffield is led by a former EPSRC Research Software Engineering Fellow and is guided by a steering panel of senior university staff who are committed to supporting the role of RSEs within the research community.
```

**Review by bobturneruk — APPROVED** [review]
Some minor wording things, if you like.

**Pull request merged**